pkg/dashboard/templates/namespace_list.gohtml

```diff
@@ -8,41 +8,67 @@
   <header class="layoutSidebar__sidebar">
     {{ template "navbar" . }}
 
-    {{ if gt (len .Data) 1 }}
+    {{ if gt (len .Data.Name) 1 }}
```

Comment:
I believe this was intended to prevent the filter from showing up if there was only one namespace, not if the name was of len>1.  Not sure why this was changed.

```suggestion
    {{ if gt (len .Data) 1 }}
```